packages/fluent-vue/src/vue/component.ts

```diff
@@ -19,6 +19,7 @@ export default defineComponent({
     path: { type: String, required: true },
     tag: { type: String, default: 'span' },
     args: { type: Object, default: () => ({}) },
+    useTa: { type: Boolean, default: false },
```

Comment:
Makes sense.

Reply:
`formatAttrs` only runs formatting if fluent message actually has attributes.
https://github.com/Demivan/fluent-vue/blob/1edd58ec1fa841e4bb3bb77799286ac914f35aa4/packages/fluent-vue/src/TranslationContext.ts#L72
So overhead will be only for getting fluent `message` from a bundle using `key`: https://github.com/Demivan/fluent-vue/blob/1edd58ec1fa841e4bb3bb77799286ac914f35aa4/packages/fluent-vue/src/TranslationContext.ts#L64-L65
This can be mitigated adding another method to `TranslationContext` that is formatting message and its attributes and returns both in one call.

Reply:
This method can actually be used to simplify directive code, as it needs formatted message with attributes too.

Reply:
Something like this?

https://github.com/davidrios/fluent-vue/commit/56c3d56dd5f1389f15b5755f179497776da7c24f

Reply:
Yes, something like that. I did some cleanup you can integrate into your branch here: https://github.com/Demivan/fluent-vue/commit/2e324e76c39e0b9e0d19644bfc93fab573be807e:

- Made methods private to not pollute `TranslationContext` type.
- Added null return to `formatWithAttrs` to distinguish between missing translation and translation without message (just with attributes)